data/presets/natural/bare_rock.json

```diff
@@ -1,5 +1,8 @@
 {
     "icon": "temaki-boulder3",
+    "moreFields": [
+        "geological_volcanic_lava_field"
```

Comment:
For the `geological=volcanic_lava_field` tag the combined usage with `natural=bare_rock` seems to be quite popular. Perhaps it could make sense to have this combination as a dedicated preset, so that it can be more easily found when mapping such features?